app/namespace.go

```diff
@@ -829,6 +829,63 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				},
 			},
 		},
+		{
+			Name:    "update-codec-server",
+			Usage:   "Update codec server config used to decode your encoded payloads through remote endpoint.",
+			Aliases: []string{"ucs"},
+			Flags: []cli.Flag{
+				NamespaceFlag,
+				&cli.StringFlag{
+					Name:     "endpoint",
+					Usage:    "The codec server endpoint to decode payloads for all users interacting with this Namespace.",
```

Comment:
[Nit] I think usage fields typically don't end with punctuation, at least with respect to the rest of the file, right?

Reply:
You're right, changed it to match the rest